plugins/outputs/timestream/timestream.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/timestreamwrite"
 	"github.com/aws/aws-sdk-go-v2/service/timestreamwrite/types"
 	"github.com/aws/smithy-go"
-
```

Comment:
Can we please keep this newline to separate 3rd party imports from telegraf internal ones?

Reply:
Makes sense

Reply:
It's still removed... :-)

Reply:
I just pushed the changes, thank you